{{cookiecutter.project_name}}/.pre-commit-configt.yaml

```diff
@@ -0,0 +1,53 @@
+# For more on configuring pre-commit hooks (see https://pre-commit.com/)
```

Comment:
can you fix the naming of this file?

Reply:
nevermind! I see you fixed this in https://github.com/dbt-labs/dbt-database-adapter-scaffold/pull/23